package.json

```diff
@@ -19,15 +19,12 @@
     "examples": "webpack-dev-server --config examples/webpack.config.js --no-info --content-base examples/"
   },
   "license": "MIT",
-  "peerDependencies": {
-    "react": "0.12.x"
-  },
   "dependencies": {
+    "react": "0.12.x",
```

Comment:
Can you give me a little more background into this being a real dependency? Do we need to switch to target any particular builder (e.g. webpack)?